core/App/lib_jsoncpp/json/features.h

```diff
@@ -0,0 +1,42 @@
+#ifndef CPPTL_JSON_FEATURES_H_INCLUDED
```

Comment:
add license header

Reply:
ignore it. due to it's comes the external git.

Reply:
The jsoncpp will be remove.